pkgs/development/python-modules/simplification/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-scm,
+  numpy,
+  cython,
+  rdp,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "simplification";
+  version = "0.7.12";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "urschrei";
+    repo = "simplification";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DgPzcpg6x7kkNfOGypbdVys5iYMyLlf+SjOE1Mr7A+A=";
+  };
+
+  prePatch = ''
+    cp ${rdp}/{include/header.h,lib/librdp.so} src/simplification
```

```suggestion
    ln -s ${rdp}/{include/header.h,lib/librdp.so} src/simplification
```
Why doesn't it suffice to put rdp in buildInputs?

---

Why `ln -s` would be better?

---

We don't want to have librdp in two places.

---

this don't change anything really...

---

What if another package starts depending on librdp?

---

this is only a build time requirement